frontend/src/component/project/Project/Import/PulsingAvatar.tsx

```diff
@@ -0,0 +1,21 @@
+import { alpha, Avatar, styled } from '@mui/material';
+
+export const PulsingAvatar = styled(Avatar, {
+    shouldForwardProp: prop => prop !== 'active',
+})<{ active: boolean }>(({ theme, active }) => ({
+    width: '80px',
+    height: '80px',
+    transition: 'background-color 0.5s ease',
+    backgroundColor: active
+        ? theme.palette.primary.main
+        : theme.palette.tertiary.main,
+    '@keyframes pulse': {
+        '0%': {
+            boxShadow: `0 0 0 0px ${alpha(theme.palette.primary.main, 0.7)}`,
+        },
+        '100%': {
+            boxShadow: `0 0 0 20px ${alpha(theme.palette.primary.main, 0.0)}`,
+        },
+    },
+    animation: active ? 'pulse 2s infinite' : '',
```

Comment:
this is the core of the animation: pulsing backgroundColor like box-shadow between alpha 0.7 (some transparency) and 0.0 (fully transparent)